.azure/templates/build-config-wpa.yml

```diff
@@ -0,0 +1,30 @@
+# This template contains steps to build MsQuic WPA plugin.
+
+jobs:
+- job: build_wpa
+  displayName: Any CPU
```

Comment:
Does this need to be `Any CPU`?  Wouldn't something like `Build` be better?

Reply:
Yeah, so it matches everything else:

![image](https://user-images.githubusercontent.com/20663557/98156372-93de3480-1e8c-11eb-9184-9d6041f067ea.png)